### ISSUE TITLE
0.16.0 does not start on Debian

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#9112]
**Describe the issue you encountered:** 0.16.0 does not start on Debian.

````bash
test@debian:~$ brave
[4636:4636:0529/161804.708893:ERROR:sandbox_linux.cc(343)] InitializeSandbox() called with multiple threads in process gpu-process. 
Crash reporting enabled
Failed to get crash dump id.
Report Id: OK
Illegal instruction
test@debian:~$
````

- Platform (Win7, 8, 10? macOS? Linux distro?): Debian

- Brave Version (revision SHA): 0.16.0

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

**Comment by arjan-s** [comment]
Experiencing the same issue on Arch Linux (package: https://aur.archlinux.org/packages/brave-bin)

**Comment by luixxiul** [comment]
@arjan-s thanks for confirming!

**Comment by Jacalz** [comment]
Same report on manjaro linux from community:
https://community.brave.com/t/fails-to-start-core-dumped-after-upgrade-to-0-16-on-linux-manjaro/3307

**Comment by luixxiul** [comment]
Fixed with 0.16.4
